src/Discord.Net.Core/Entities/Interactions/MessageComponents/IComponentInteractionData.cs

```diff
@@ -18,12 +18,32 @@ public interface IComponentInteractionData : IDiscordInteractionData
         ComponentType Type { get; }
 
         /// <summary>
-        ///     Gets the value(s) of a <see cref="SelectMenuComponent"/> interaction response.
+        ///     Gets the value(s) of a <see cref="ComponentType.SelectMenu"/> interaction response. Null if select type is different.
         /// </summary>
         IReadOnlyCollection<string> Values { get; }
 
         /// <summary>
-        ///     Gets the value of a <see cref="TextInputComponent"/> interaction response.
+        ///     Gets the channels(s) of a <see cref="ComponentType.ChannelSelect"/> interaction response. Null if select type is different.
```

Comment:
```suggestion
        ///     Gets the channels(s) of a <see cref="ComponentType.ChannelSelect"/> interaction response. <see langword="null"/> if select type is different.
```

Reply:
Minor nitpick, but please use keyword references instead.

Reply:
*oops*, forgot this existed. Should be fixed now *for real this time*